apps/service_providers/llm_service/main.py

```diff
@@ -252,12 +252,8 @@ class OpenAIGenericService(LlmService):
     openai_api_key: str
     openai_api_base: str
 
-    def get_chat_model(self, llm_model: str, temperature: float, **kwargs) -> BaseChatModel:
-        model = ChatOpenAI(
-            model=llm_model,
-            temperature=1 if llm_model.startswith(("o3", "o4", "gpt-5")) else temperature,
-            **self._get_model_kwargs(**kwargs),
-        )
+    def get_chat_model(self, llm_model: str, **kwargs) -> BaseChatModel:
+        model = ChatOpenAI(model=llm_model, **self._get_model_kwargs(**kwargs), use_responses_api=True)
```

Comment:
Adding `use_responses_api=True` seems unrelated to this PR. Was there a reason for adding it?

Possibly relevant other ticket: https://github.com/dimagi/open-chat-studio/issues/2438